exclude_list.yaml

```diff
@@ -25,6 +25,20 @@
 - package_name: 'gdbgui'
   version: '==0.13.2.0'
 
-# cffi==1.17.0 does not exist yet it is required by some wheel (7/11/2024)
-- package_name: 'cffi'
-  version: '==1.17.0'
+# Python 13.3 does not support Pillow 9.5.0
+- package_name: 'Pillow'
+  version: '==9.5.0'
+  python: '==3.13'
+
+# gdbgui leads to installation of greenlet which does not support Python 3.13 yet (10/9/2024)
+- package_name: 'gdbgui'
```

Comment:
Please add the links I've listed here as comments.

Reply:
Sorry, now it should be as you meant it.

Reply:
@dobairoland, are you OK with merging this now?

The Python 3.13 wheels are already uploaded on the Espressif's server because of the performed tests so there is no need to rush with it but it will be better to have them built periodically and get back to it when the not supported packages will be updated.